src/Login/index.tsx

```
@@ -0,0 +1,134 @@
+          >
+            Email
+          </FormControl.Label>
+          <Input
+            autoCapitalize="none"
+            value={email}
+            onChangeText={(inputValue) => setEmail(inputValue)}
+          />
+        </FormControl>
+        <FormControl mb={5} isInvalid={!!errorMessage}>
+          <FormControl.Label
+            _text={{ color: 'muted.700', fontSize: 'sm', fontWeight: 600 }}
+          >
+            Password
+          </FormControl.Label>
+          <Input
+            type={show ? 'text' : 'password'}
+            value={password}
+            onChangeText={(inputValue) => setPassword(inputValue)}
+            InputRightElement={
+              <Button
+                ml={1}
+                roundedLeft={0}
+                roundedRight="md"
+                onPress={handlePasswordShow}
+              >
+                {show ? 'Hide' : 'Show'}
+              </Button>
+            }
+          />
+          <FormControl.ErrorMessage
+            _text={{ fontSize: 'xl', color: 'error.500', fontWeight: 500 }}
+          >
+            {errorMessage ? errorMessage : null}
+          </FormControl.ErrorMessage>
+        </FormControl>
+        <VStack space={2}>
+          <Button
+            colorScheme="cyan"
+            _text={{ color: 'white' }}
+            onPress={handleLogin}
+            isLoading={isLoading}
+          >
+            Login
+          </Button>
+        </VStack>
+      </VStack>
+    </Box>
+  );
+};
+
+const login = async (email: string, password: string) => {
+  try {
+    const response = await fetch(`${API_URL}/auth/login`, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({
+        username: email,
+        password,
+      }),
+    });
+
+    if (response.status >= 200 && response.status < 300) {
+      return await response.json();
+    }
+    return false;
```

you should throw an Error instead